src/component/association-select.js

```diff
@@ -176,14 +184,18 @@ export class AssociationSelect {
     return this;
   }
 
+  resourceChanged(resource) {
+    if (!resource) {
+      logger.error('resource bindable should be a string instead of ' + typeof resource);
+    }
+
+    this.repository = this.entityManager.getRepository(resource);
```

Comment:
If `resource` has to be a string, why don't you just check `if (typeof resource === 'string')` instead of `if (!resource)`?